fragment.go

```diff
@@ -7,9 +7,10 @@ import (
 )
 
 type Fragment struct {
-	Path     string `json:"path"`
-	Url      string
-	Metadata map[string]string `json:"metadata"`
+	Path        string `json:"path"`
+	Url         string
+	Metadata    map[string]string `json:"metadata"`
+	TimingLabel string
```

Comment:
> Do you think it would make sense to make this configurable via the JSON file, or maybe we could default to Path if a TimingLabel isn't provided?

Yeah, thanks, that was an oversight, I do want it configurable via JSON, will update.

I initially went with a new field only to make this opt-in, but happy to have it fallback to `Path` if you'd prefer.

I didn't include it in metadata because it didn't seem like something we'd always want coupled to the tracing tags.

Reply:
> I initially went with a new field only to make this opt-in, but happy to have it fallback to Path if you'd prefer.

Looks like path might not work since `/` characters aren't allowed in server timing token headers:

https://w3c.github.io/server-timing/#the-server-timing-header-field
https://httpwg.org/specs/rfc7230.html#rfc.section.3.2.6

So I think I'd like to go with an explicit label to start if that is okay with you.

Reply:
Yep, sounds good! 👍